microprofile-lra/README.adoc

```diff
@@ -0,0 +1,592 @@
+* xref:start_the_eap_standalone_server[Start the {productName} server with the standalone default profile] as described above.
+
+. Review the `enable-microprofile-lra.cli` file in the root of this quickstart directory. It enables two extensions and adds
+two subsystems, one for LRA coordinator and one for LRA participant respectively.
+. Open a new terminal, navigate to the root directory of this quickstart, and run the following command, replacing `__{jbossHomeName}__`
+with the path to your server:
++
+[source,subs="+quotes,attributes+",options="nowrap"]
+----
+$ __{jbossHomeName}__/bin/jboss-cli.sh --connect --file=enable-microprofile-lra.cli
+----
++
+NOTE: For Windows, use the `__{jbossHomeName}__\bin\jboss-cli.bat` script.
++
+
+You should see the following result when you run the script:
++
+[source,options="nowrap"]
+----
+The batch executed successfully
+----
+
+. Stop the {productName} server.
+
+== Review the Modified Server Configuration
+
+After stopping the server, open the `__{jbossHomeName}__/standalone/configuration/{configFileName}` file and review the changes.
+
+. The script added the following two extensions:
++
+[source,xml,options="nowrap"]
+----
+<extension module="org.wildfly.extension.microprofile.lra-coordinator"/>
+<extension module="org.wildfly.extension.microprofile.lra-participant"/>
+----
++
+
+. And also the following two subsystems:
++
+[source,xml,options="nowrap"]
+----
+<subsystem xmlns="urn:wildfly:microprofile-lra-coordinator:1.0"/>
+<subsystem xmlns="urn:wildfly:microprofile-lra-participant:1.0"/>
+----
++
+
+[[solution]]
+== Solution
+
+We recommend that you follow the instructions that
+<<creating-new-project, create the application step by step>>. However, you can
+also go right to the completed example which is available in this directory.
+
+// Build and Deploy the Quickstart
+include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
+
+// Server Distribution Testing
+include::../shared-doc/run-integration-tests-with-server-distribution.adoc[leveloffset=+2]
+
+// Undeploy the Quickstart
+include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
+
+// Restore the {productName} Standalone Server Configuration
+include::../shared-doc/restore-standalone-server-configuration.adoc[leveloffset=+1]
+
+[[creating-new-project]]
+== Creating the Maven Project
+
+[source,options="nowrap"]
+----
+mvn archetype:generate \
+    -DgroupId=org.wildfly.quickstarts \
+    -DartifactId=microprofile-lra \
+    -DinteractiveMode=false \
+    -DarchetypeGroupId=org.apache.maven.archetypes \
+    -DarchetypeArtifactId=maven-archetype-webapp
+cd microprofile-lra
+----
+
+Open the project in your favourite IDE.
+
+Open the generated `pom.xml`.
+
+The first thing to do is to change the minimum JDK to Java 11:
+
+[source,xml]
+----
+<maven.compiler.source>11</maven.compiler.source>
+<maven.compiler.target>11</maven.compiler.target>
+----
```

Comment:
There are no instructions regarding 
```
<version.server>30.0.0.Final</version.server>
<version.bom.ee>${version.server}</version.bom.ee>
<version.bom.microprofile>${version.server}</version.bom.microprofile>
```

Reply:
Agree.
PS: This is why I dislike "create from scratch" on readmes, it will have a hard time catching up with sources.

Reply:
Less work for us, more investigations for the user/customer. I still think it's worth having step by step instructions.

Reply:
For this we could also say copy the pom.xml from the QS but since the QS is not independent of the parent properties this is not possible.